cms/schemas/Happening.js

```diff
@@ -131,6 +131,24 @@ export default {
             title: 'Åpen kun for studentgrupper?',
             type: 'boolean',
         },
+        {
+            name: 'deductible',
+            title: 'Må deltakeren betale egenandel?',
+            initialValue: false,
+            type: 'boolean',
+        },
+        {
+            name: 'deductiblePayment',
+            title: 'Hva må deltakeren betale i egenandel?',
+            type: 'string',
+            hidden: ({ document }) => !document?.deductible,
```

Comment:
Greit å passe på at feltet ikke har noen verdi når det er hidden

```suggestion
            hidden: ({ document, value }) => !value && !document?.deductible,
```

Reply:
Lurt 👍